cfgov/cal/views.py

```diff
@@ -10,17 +10,19 @@
 from rest_framework import generics
 
 from django.shortcuts import render, render_to_response
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 from django import forms
 from django.utils.safestring import mark_safe
```

Comment:
this import as well as a bunch of others are never used.

Here's the list:

date, datetime
from dateutil.relativedelta import relativedelta
from rest_framework import generics
from django.utils.safestring import mark_safe
from django.template import RequestContext
from django.core.exceptions import ObjectDoesNotExist